Dockerfile

```diff
@@ -51,18 +52,26 @@ RUN set -x \
    && rm tini-static.asc \
    && mv tini-static tini \
    && chmod +x tini \
+
    # Install Habitat
-   && wget 'https://api.bintray.com/content/habitat/stable/linux/x86_64/hab-$latest-x86_64-linux.tar.gz?bt_package=hab-x86_64-linux' \
-   && tar -C . -ozxvf hab-\$latest-x86_64-linux.tar.gz?bt_package=hab-x86_64-linux \
-   && mkdir /opt/sd/bin \
-   && mv /opt/sd/hab-*-x86_64-linux/hab /opt/sd/bin/hab \
-   && chmod +x /opt/sd/bin/hab \
-   && rm -rf hab-* \
+   && mkdir -p /hab/bin /opt/sd/bin \
+   # Download Habitat Binary
+   && wget -O hab.tar.gz 'https://api.bintray.com/content/habitat/stable/linux/x86_64/hab-$latest-x86_64-linux.tar.gz?bt_package=hab-x86_64-linux' \
+   && tar -C . -ozxvf hab.tar.gz \
+   && mv hab-*/hab /hab/bin/hab \
+   && chmod +x /hab/bin/hab \
+   # @TODO Remove this, I don't think it belongs here.  We should use /hab/bin/hab instead.
+   && cp /hab/bin/hab /opt/sd/bin/hab \
```

Comment:
TODO comment +1
note: We need to fix [this](https://github.com/screwdriver-cd/sd-step/blob/master/sd-step.go#L26) once new launcher is released